docs/modules/ROOT/pages/vshn-managed/keycloak/customization.adoc

```diff
@@ -0,0 +1,29 @@
+= Customization
+
+This page will describe how you can customize your Keyclock instance.
+
+== Themes and providers
+
+To customize providers and themes customization images parameter should be used.
+
+Include your own image with custom themes and providers. Place your themes in the `/themes` directory and your providers in the `/providers` directory within your custom Docker image.
```

Comment:
I feel like there's some context missing here for people who don't know how this is supposed to work. Maybe go into more detail. Like an introduction that a custom image is needed to add the customizations or how the customization works.

Because right now it's not really clear what to do, if you've never used this system for customization.

Reply:
I'm with Simon, I have no idea what we're talking about here. Put yourself into the position of an end-user, not having any idea about how our system works, with that hat on, it lacks a lot of context.